crane/search/base.py

```diff
@@ -92,7 +110,7 @@ def _get_data(url):
         """
         try:
             # one second timeout
-            response = urllib2.urlopen(url, timeout=1)
+            response = urllib2.urlopen(url, timeout=30)
```

Comment:
any reason you bumped to such a bit timeout in comparison the the old one?

Reply:
The Solr takes more than 1 second in some cases. resulting in 503. Hence. I had bumped it up.